app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -610,6 +618,42 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_hasCorrectActivityLabel_adminControls() {
+    activityTestRule.launchActivity(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context = context,
+        adminPin = "12345",
+        profileId = internalProfileId,
+        colorRgb = -10710042,
+        adminPinEnum = AdminAuthEnum.PROFILE_ADMIN_CONTROLS.value
+      )
+    )
+    val title = activityTestRule.activity.title
+
+    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+    // correct string when it's read out.
+    assertThat(title).isEqualTo(context.getString(R.string.admin_auth_activity_access_controls_title))
```

Comment:
This should be broken into 2 lines.
Basically any line in android studio should not cross more than 100 chars.

You can check the error in detail here: https://github.com/oppia/oppia-android/pull/2904/checks?check_run_id=2115400132

Reply:
sorry about that I forgot to check linting issues, fixed it!

Reply:
No issues. Thanks.